docs/sources/reference/run.md

```diff
@@ -552,6 +553,20 @@ would be 2*300M, so processes can use 300M swap memory as well.
 We set both memory and swap memory, so the processes in the container can use
 300M memory and 700M swap memory.
 
+We can set oom control in which whether to disable OOM Killer or not when OOM
+happened. It is strongly recommended that use this option with `-m`.
```

Comment:
I hope I used all the terms right, but after reading the related discussions, I think it should be okay :)